app/bundles/InstallBundle/Configurator/Form/DoctrineStepType.php

```diff
@@ -31,7 +36,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder->add(
             'driver',
-            'choice',
+            ChoiceType::class,
             [
                 'choices'     => DoctrineStep::getDrivers(),
```

Comment:
All the choice types needs the option `choices_as_values` and flipped labels with values as described in https://docs.google.com/document/d/1lHNK8hHQtvyWlz9OtfAoLupDVb5-QunQjq-HAGlAsS4/edit#